simapp/simd/cmd/root.go

```diff
@@ -74,9 +74,12 @@ func Execute(rootCmd *cobra.Command) error {
 	// and a Tendermint RPC. This requires the use of a pointer reference when
 	// getting and setting the client.Context. Ideally, we utilize
 	// https://github.com/spf13/cobra/pull/1118.
+	srvCtx := server.NewDefaultContext()
 	ctx := context.Background()
 	ctx = context.WithValue(ctx, client.ClientContextKey, &client.Context{})
-	ctx = context.WithValue(ctx, server.ServerContextKey, server.NewDefaultContext())
+	ctx = context.WithValue(ctx, server.ServerContextKey, srvCtx)
+
+	rootCmd.PersistentFlags().String("log_level", srvCtx.Config.LogLevel, "The logging level in the format of <module>:<level>,...")
```

Comment:
Maybe we could add `"log_level"` to client/flags/flags.go, like other flags?

Reply:
The reason I didn't is because the SDK doesn't actually use this flag at all, it's parsed by Tendermint when we handle config interception/parsing.

Reply:
Can we replace the underscore character `_` with a `-`? It looks ugly and a bit inconsistent with the rest of the flagset.

Reply:
Late addition: owh true, Tendermint parses it.. so we may not have a choice and keep it in the ugly form. In that case, nevermind and press ahead...

Reply:
No, again, this is a Tendermint flag/config. It must match what's in Tendermint which is `log_level`.